packages/jaeger-ui-components/src/common/CopyIcon.tsx

```diff
@@ -78,20 +80,18 @@ export default class CopyIcon extends React.PureComponent<PropsType, StateType>
   render() {
     const styles = getStyles();
     return (
-      <UITooltip
-        arrowPointAtCenter
-        mouseLeaveDelay={0.5}
-        onVisibleChange={this.handleTooltipVisibilityChange}
+      <Tooltip
+        // onVisibleChange={this.handleTooltipVisibilityChange}
```

Comment:
I know there's a big TODO on top of this component but Is this a leftover?

Reply:
Yes. This component was not showing before and the deeplinking feature it has is not working right now. This will be fixed in a separate pr.